lib/RGBot.js

```diff
@@ -496,13 +511,109 @@ class RGBot extends EventEmitter {
      * Pauses the bot to wait for the desired number of milliseconds. Use this
      * function to make the bot wait for a certain amount of time, instead
      * of a certain number of ticks (see `wait(ticks: number)`).
+     * If calling from python this will wait the number of ticks at least as long as the desired
+     * number of milliseconds. (50ms = 1 tick, 55ms = 2 ticks, 99ms = 2 ticks, 100ms = 2 ticks, 101ms = 3ticks)
+     * This behavior is because the python/js bridge can't support timers/intervals.
      * @see wait
      * @param {number} milliseconds The number of milliseconds to wait
      * @return {Promise<void>}
      */
     async waitForMilliseconds(milliseconds) {
         if (milliseconds < 0) return;
-        return new Promise(resolve => setTimeout(resolve, milliseconds));
+        if (!this.#isPython) {
+            return new Promise(resolve => setTimeout(resolve, milliseconds));
+        } else {
+            // python can't use timers, wait ticks instead
+            const toWait = milliseconds/50
+            // if not an even 50, tack on 1 tick
+            const extra = milliseconds%50 > 0 ? 1 : 0
+            await this.wait( toWait+extra)
```

Comment:
shouldn't this use the setTimout  method you defined below so there's no change to this method?

Reply:
changing for consistency